frontend/src/components/useQuery.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import { useLocation } from "react-router-dom";
+
+export default function useQuery() {
```

Comment:
Thanks! Could you move it under `/utils` and maybe create a file called `hooks.tsx` to store all the custom hooks?

Reply:
I saw a few other custom hooks under `components/`. How about I create a `/hooks` folder and move all of them there? I.e., we will end up having `/hooks/useEnv.tsx`, `/hooks/useQuery.tsx`, ...

I think it is probably better to create another PR if we want to do this.